### ISSUE TITLE
Problems to detect no primitive field types using the NestJS GraphQL plugin with the Code-First approach

### ISSUE THREAD

**Issue by penguintheorem** [nestjs/graphql#725]
## Current behavior

I've recently upgraded a GraphQL API built with NestJS to the version 7.

[I've setup the project](https://docs.nestjs.com/graphql/resolvers#cli-plugin) to use the nest-cli GrapQL plugin with the code first approach, however i got the following error at runtime:

```
(node:22783) UnhandledPromiseRejectionWarning: Error: Cannot determine GraphQL output type for auth
    at OutputTypeFactory.create (/Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/@nestjs/graphql/dist/schema-builder/factories/output-type.factory.js:17:23)
    at /Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/@nestjs/graphql/dist/schema-builder/factories/object-type-definition.factory.js:60:53
    at Array.forEach (<anonymous>)
    at /Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/@nestjs/graphql/dist/schema-builder/factories/object-type-definition.factory.js:59:33
    at resolveThunk (/Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/graphql/type/definition.js:438:40)
    at defineFieldMap (/Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/graphql/type/definition.js:625:18)
    at GraphQLObjectType.getFields (/Users/attylancer/Documents/career/customer-projects/stamp/stamp-graphql/node_modules/graphql/type/definition.js:579:27)
```

## Input Code

[Here](https://github.com/penguintheorem/nest-graphql-test.git) a minimal repository that reproduces the issue.

You should be able to reproduce the issue by running:

`npm i && npm run start` or `npm i && npm run start:dev`

## Expected behavior

As mentioned [here](https://trilon.io/blog/announcing-nestjs-7-whats-new#NestJS-GraphQL-Plugin), we'd like to avoid to put the decorator `@Field()` on each no primitive fields of every `@ObjectType()` marked class.

## Environment
```
Nest version: 7.0.3
Node version: 12.9.1
Platform: Mac
```

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by penguintheorem** [comment]
> Please, provide a minimal repository which reproduces your issue.

done ! let me know if you need further details, thanks

**Comment by kamilmysliwiec** [comment]
Fixed in 7.1.5. Update your package + remove dist + remove tsconfig build cache + rebuild :)
